### PR TITLE
Added ICRA 2019 paper on "Sensing Shear Forces During Food Manipulation"

### PR DESCRIPTION
**Pull request by hanjuns**
- [x] Update `.bib` file
- [x] Upload publication PDF to [Google Drive](https://drive.google.com/drive/folders/1M9fOGIIQ3e1R62dtVit5rZ5iWZqxfWV9)

### CODE REVIEW

File: siddpubs-conf.bib [hunk 1]
```diff
@@ -52,6 +52,13 @@ @string{icml
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{hanjun2019shear,
+  title={Sensing Shear Forces During Food Manipulation: Resolving the Trade-Off Between Range and Sensitivity},
+  author={Hanjun Song, Tapomayukh Bhattacharjee, and Siddhartha S. Srinivasa},
```
Comment (brianhou) [inline siddpubs-conf.bib]: Please use initials rather than first names (see below for examples).

**Review by brianhou — CHANGES_REQUESTED** [review]

File: siddpubs-conf.bib [hunk 2]
```diff
@@ -52,6 +52,13 @@ @string{icml
 
 % 2019 %%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
 
+@inproceedings{hanjun2019shear,
```
Comment (brianhou) [inline siddpubs-conf.bib]: Please use your last name as the key, i.e. `song2019shear`.

File: siddpubs-conf.bib (hunk 1 above)
Comment (hanjuns) [reply siddpubs-conf.bib]: ok, done!

File: siddpubs-conf.bib (hunk 2 above)
Comment (hanjuns) [reply siddpubs-conf.bib]: ok, done!

**Review by brianhou — APPROVED** [review]

**Pull request merged**